intro-to-R-tidyverse/02-intro_to_ggplot2.Rmd

```diff
@@ -34,7 +34,7 @@ We performed three sets of contrasts:
 - [ggplot2 website](https://ggplot2.tidyverse.org/)
 - [Handy cheatsheet for ggplot2 (pdf)](https://github.com/rstudio/cheatsheets/raw/main/data-visualization.pdf)
 - [_Data Visualization, A practical introduction_](https://socviz.co/)
-- [Data visualization chapter of _R for Data Science_](https://r4ds.had.co.nz/data-visualisation.html)
+- [Data visualization chapter of _R for Data Science_](https://r4ds.hadley.nz/data-visualize)
```

Comment:
Whatever you do in https://github.com/AlexsLemonade/exercise-notebook-answers/pull/210, do the same here (w/ or w/o `.html`)